achyut/src/main/java/com/lftechnology/runner/Executer.java

```diff
@@ -0,0 +1,25 @@
+package com.lftechnology.runner;
+
+import java.util.Scanner;
+
+public class Executer {
+
+    private static Scanner userInput;
+    public static int chosenOption;
+
+    public static void main(String[] args) {
+        Menu menu = new Menu();
+        menu.showmenu();
+    }
+
+    public void askForInputAndRun() {
+        /**
+         * @todo Check if user input is empty in all the subsequent classes
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1135)